src/js/components/Select/__tests__/__snapshots__/Select-test.js.snap

```diff
@@ -2412,7 +2412,7 @@ exports[`Select basic 1`] = `
 }
 
 <button
-  aria-label="Open Drop"
+  aria-label="Select"
```

Comment:
Why did this one change to Select?

Reply:
This is this test 

```
  test('basic', () => {
    const component = renderer.create(
      <Select id="test-select" options={['one', 'two']} a11yTitle="Select" />,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });

```

The allytitle was set to `select` so that's correct now